src/lib/y2partitioner/widgets/delete_disk_partition_button.rb

```diff
@@ -24,18 +46,17 @@ def initialize(device: nil, table: nil, device_graph: nil)
         @device_graph = device_graph
       end
 
+      # @macro seeAbstractWidget
       def label
         _("Delete...")
```

Comment:
Add a comment for translators to get the context of the text, e.g.

``` ruby
# TRANSLATORS: push button label
```

It might be tricky to translate it correctly...